tools/wptrunner/wptrunner/browsers/content_shell.py

```diff
@@ -43,6 +43,10 @@ def check_args(**kwargs):
 
 
 def browser_kwargs(logger, test_type, run_info_data, config, subsuite, **kwargs):
+    ENABLE_THREADED_COMPOSITING_FLAG = '--enable-threaded-compositing'
```

Comment:
nit: The name style implies a constant, which should usually be [at the module level](https://peps.python.org/pep-0008/#constants).

Reply:
Ack